### ISSUE TITLE
Order Manager doesn't display graphql error messages

### ISSUE THREAD

**Issue by eliotjordan** [pulibrary/figgy#5370]
Example graphql response with an error.

```
data: {input: {,…}}}
data: {input: {,…}}
input: {,…}
errors: ["Member ids can only be used to re-order."]
0: "Member ids can only be used to re-order."
resource: {id: "9f57a91e-08ed-4e0a-9662-0d6670fb848b", thumbnail: {id: "2d9e1835-9003-4b55-9c93-de91b2b08b63",…},…}
id: "9f57a91e-08ed-4e0a-9662-0d6670fb848b"
members: [{id: "2d9e1835-9003-4b55-9c93-de91b2b08b63", label: "front cover",…},…]
[0 … 99]
```

Here's an example graphql error in honeybadger:
`Valkyrie::Persistence::StaleObjectError in graphql # execute`
https://app.honeybadger.io/projects/53391/faults/95511387

**Comment by eliotjordan** [comment]
Probably an issue with the structure manager as well.

**Comment by hackartisan** [comment]
Do we know how to replicate this?

**Comment by sdellis** [comment]
Duplicate of #4847

**Comment by sdellis** [comment]
@tpendragon here is the screenshot of the new error message in the Order Manager. Working on a Vue test for this, but I'm currently blocked by an error from Circle CI that it can't compile webpacker.

![Screen Shot 2022-10-10 at 2.35.19 PM.png](https://images.zenhubusercontent.com/5cba15f2e89c9c3b0c58e0d1/943f19f7-8884-4265-90db-260babdbd49c)

**Comment by sdellis** [comment]
We need to test [these lines of code](https://github.com/pulibrary/figgy/blob/i5370-graphql-error-alerts/app/javascript/store/vuex/actions.js#L88-L89). 

This type of test where we [test the commit and check the state for an update](https://github.com/pulibrary/figgy/blob/main/app/javascript/test/resource.spec.js#L83-L90), could potentially work.

We could also potentially [test the Vuex Action](https://vuex.vuejs.org/guide/testing.html#testing-actions) itself. Or we could do [a feature test on the Order Manager](https://github.com/pulibrary/figgy/blob/i5370-graphql-error-alerts/spec/features/order_manager_spec.rb).

Most of these require knowing how to break the test resource so that it throws an error when saving, which I'm not sure how to do.
